ctapipe/instrument/subarray.py

```diff
@@ -56,13 +56,15 @@ class SubarrayDescription:
     ----------
     name: str
        name of subarray
-    tel_coords: astropy.coordinates.SkyCoord
+    tel_coords: ctapipe.coordinates.GroundFrame
```

Comment:
Is this a pure documentation update? Actual code changes in this PR seems to not react to this change, so was the input always supposed to be in `GroundFrame`?

Reply:
Both a SkyCoord in GroundFrame and a GroundFrame directly work, however internally, we use GroundFrame as 

a) it is slightly faster
b) Calling something on the ground a SkyCoord doesn't really make sense

Also see #1278